docs/public-networks/concepts/node-sync.md

```diff
@@ -21,141 +25,93 @@ Full nodes can guarantee the latest state of the blockchain (and some older stat
 can't serve the network with all data requests (for example, the balance of an account at an old 
 block).
 
+You can run a full node using [snap synchronization](#snap-synchronization),
+[checkpoint synchronization](#checkpoint-synchronization), or
+[fast synchronization](#fast-synchronization).
+
+### Archive nodes
+
 An archive node is a node that also stores the intermediary state of every account and contract
 for every block since the genesis block. 
-
 Archive nodes can do everything full nodes do, and they can also access historical state data. 
 This means that archive nodes require more disk space than full nodes.
 
-Besu must connect with other peers to sync with the network. If your node is having trouble peering, 
-try [troubleshooting peering](../how-to/troubleshoot/peering.md).
-
-## Sync modes
-
-Besu supports several sync modes for different network types and use cases.
-For public networks, Besu offers the following sync modes:
-
-| Sync mode | Description | Requirements | Limitations |
-|-----------|-------------|--------------|-------------|
-| [Snap](#snap-synchronization) | Default for Mainnet. Efficient sync from genesis block, downloading as many trie leaves as possible and reconstructing locally. Faster than fast sync. | Besu version 22.4.0 or later | Cannot switch from fast sync to snap sync mid-process. |
-| [Checkpoint](#checkpoint-synchronization) | Syncs from a specific checkpoint block configured in the genesis file. Fastest sync mode with lowest storage requirements. | Besu version 22.4.3 or later | |
-| [Fast](#fast-synchronization) | Default for named networks except `dev`. Downloads block headers and transaction receipts, verifies chain from genesis block. | None | Might become impossible to fast sync Ethereum Mainnet in the future. |
-| [Full](#run-an-archive-node) | Downloads and verifies the entire blockchain and state from genesis block, building an archive node with full state history. | None | Slowest sync mode, requires the most disk space. |
-
-:::tip
-
-We recommend snap sync because it follows the Ethereum specification and enables you to serve full historical data.
+You can only run an archive node using [full synchronization](#full-synchronization).
 
+:::caution important
+Do not run an archive node with the [Bonsai Tries](data-storage-formats.md#bonsai-tries)
```

Comment:
Just for consistency with the other notes. I think we do a space before and after?

```suggestion

Do not run an archive node with the [Bonsai Tries](data-storage-formats.md#bonsai-tries)
```